src/xbrl_extract/arelle_interface.py

```diff
@@ -15,19 +15,40 @@ def load_xbrl(path: str):
     return ModelXbrl.load(model_manager, path)
 
 
+def load_taxonomy(path: str):
```

Comment:
Can you include argument descriptions in the docstrings?

Reply:
Fixed